src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -84,17 +84,18 @@ internal static IExpression CreateEvaluatorDirect(CheckResult result, StackDepth
             result.ThrowOnErrors();
 
             (var irnode, var ruleScopeSymbol) = IRTranslator.Translate(result._binding);
-            return new ParsedExpression(irnode, ruleScopeSymbol, stackMarker);
+            return new ParsedExpression(irnode, ruleScopeSymbol, stackMarker, features);
         }
 
         /// <summary>
-        /// Create an evaluator over the existing binding. 
+        /// Create an evaluator over the existing binding.
         /// </summary>
-        /// <param name="result">A successful binding from a previous call to <see cref="Engine.Check(string, RecordType, ParserOptions)"/>. </param>
+        /// <param name = "result" >A successful binding from a previous call to.<see cref="Engine.Check(string, RecordType, ParserOptions)"/>. </param>
+        /// <param name="features">Features to use.</param>
         /// <returns></returns>
-        public static IExpression CreateEvaluatorDirect(CheckResult result)
+        public static IExpression CreateEvaluatorDirect(CheckResult result, Features features = Features.None)

```

Comment:
>Features features = Features.None [](http://example.com/codeflow?start=76&length=33)

new default param  is technically binary breaking change.  (Bruce's tool should now catch this) 

For now, it should be added as a 2nd separate overload.